docs/core/deploying/index.md

```diff
@@ -1,95 +1,104 @@
+| Type                                                          | SDK 3.1 | SDK 3.0 | SDK 2.1 | Command | 
+| ------------------------------------------------------------- | --------| ------- | ------- | ------- |
+| [runtime-dependent cross-platform binary](#runtime-dependent) | ✔️       | ✔️       | ✔️       | [`dotnet publish`](../tools/dotnet-publish.md) |
+| [runtime-dependent executable](#runtime-dependent)            | ✔️       | ✔️       |          | [`dotnet publish`](../tools/dotnet-publish.md) |
+| [self-contained executable](#self-contained)                  | ✔️       | ✔️       | ✔️       | [`dotnet publish -r <RID>`](../tools/dotnet-publish.md) |
 
-- Framework-dependent executables. Produces an executable that runs on a target platform. Similar to FDDs, framework-dependent executables (FDE) are platform-specific and aren't self-contained. These deployments still rely on the presence of a shared system-wide version of .NET Core to run. Unlike an SCD, your app only contains your code and any third-party dependencies that are outside of the .NET Core libraries. FDEs produce an executable that runs on the target platform.
+## Runtime-dependent
 
-## Framework-dependent deployments (FDD)
+Apps published as runtime-dependent are cross-platform and don't include the .NET Core runtime. However, the user of your app is required to install the .NET Core runtime.
 
-For an FDD, you deploy only your app and third-party dependencies. Your app will use the version of .NET Core that's present on the target system. This is the default deployment model for .NET Core and ASP.NET Core apps that target .NET Core.
+A *dll* file is created when you publish your app. For example, if you have an app named **word_reader**, a file named *word_reader.dll* is created. Apps published in this way are run with the `dotnet <filename.dll>` command and can be run on any platform.
 
-### Why create a framework-dependent deployment?
+Along with the *dll* file containing your app, a platform-specific executable can be published if the `-r <RID>` parameter is used. The *dll* is still created and remains cross-platform while the executable isn't. Continuing the example above, if you publish the **word_reader** app and target Windows, a *word_reader.exe* executable is created along with *word_reader.dll*. When targeting Linux or macOS, a *word_reader* executable is created along with *word_reader.dll*. For more information about RIDs, see [.NET Core RID Catalog](../rid-catalog.md).
```

Comment:
Suggest removing "if the ...is used" here. Then after this and the next paragraph use samples or a little chart about what combination of switches you need - it's not just -r, but also --self-contained false. And then maybe do the "continuing the example" after the samples or table.